packages/ember-runtime/lib/mixins/array.js

```diff
@@ -1237,126 +1233,20 @@ const ArrayMixin = Mixin.create(Enumerable, {
     @public
   */
   '@each': computed(function() {
-    // TODO use Symbol or add to meta
-    if (!this.__each) {
-      this.__each = new EachProxy(this);
-    }
+    deprecate(
+      "Getting the '@each' property is deprecated",
+      false,
+      {
+        id: 'ember-metal.getting-each',
+        until: '3.5.0',
+        url: 'https://emberjs.com/deprecations/v3.x#toc_getting-the-each-property'
+      }
+    );
 
-    return this.__each;
+    return meta(this).eachProxy;
   }).volatile().readOnly()
```

Comment:
I can’t recall, why is this volatile (seems like it is stable based on the implementation)?

Reply:
I don't know exactly why it was volatile, but it certainly doesn't need to be now (in fact, we could remove this CP if we wanted).